proxy/src/proxy.rs

```diff
@@ -110,6 +110,19 @@ static COMPUTE_CONNECTION_LATENCY: Lazy<HistogramVec> = Lazy::new(|| {
     .unwrap()
 });
 
+static CONTROL_PLANE_LATENCY: Lazy<HistogramVec> = Lazy::new(|| {
```

Comment:
I think it makes sense to add slices by the requests which proxy send to the control plane.
For example `get_role_secret` is insignificantly lightweight in comparison with the `wake_compute`.
Having one more slice showing what exactly did happen would be very useful.

Reply:
That is already tracked in `proxy_console_request_latency`. The point of this is that it's for all requests that a connection attempt might make. Because of this, there's no single value I can provide the histogram here

Reply:
Not exactly, in `proxy_console_request_latency` we don't do it per request. What I was trying to say, in this implementation it's impossible to distinguish between calling just `wake_compute` or `wake_compute` and `get_role_secret`.

`proxy_console_request_latency` gives you a visibility on how slow control plane is in general. But it doesn't tell you how it impacts the connections per protocol/per outcome etc.